tests/test_utils.py

```diff
@@ -603,6 +605,23 @@ def load_data_into_test_database(
             dataframe=all_dfs_concatenated, table_name=table_name
         )
         return return_value
+    elif engine.dialect.name.lower() == "awsathena":
+        try:
+            connection = engine.connect()
+            load_dataframe_into_test_athena_database_as_table(
+                df=all_dfs_concatenated,
+                table_name=table_name,
+                connection=connection,
+            )
+            return return_value
+        except SQLAlchemyError as e:
+            logger.error(
+                """Docs integration tests encountered an error while loading test-data into test-database."""
+            )
+            raise
```

Comment:
Will this ever log the connection string? Should we be concerned about security at all?

Reply:
This is a really good question. It does. I just checked. For example: `sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "test_ci_invalid" does not exist.` Let's discuss with the team.